serde-api/src/main/java/io/micronaut/serde/config/annotation/SerdeConfig.java

```diff
@@ -266,10 +271,17 @@
      */
     @Internal
     @interface SerSubtyped {
+
         /**
          * @return the subtypes
          */
         SerSubtype[] value() default {};
+
+        /**
+         * Is discriminator visible.
+         */
+        String DISCRIMINATOR_VISIBLE = "dvisible";
```

Comment:
```suggestion
        String DISCRIMINATOR_VISIBLE = "discriminatorVisible";
```

short name is confusing imo

Reply:
I was trying to follow the style somebody established :D

Reply:
oh yea, the other's aren't great either imo :sweat_smile: but too late for those